### PR TITLE
small improvements to make bundle smaller

### PR DESCRIPTION
**Pull request by eran10**
work with specific lodash packages to make the size of the package smaller

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "audit-service",
-  "version": "0.1.3",
+  "version": "0.1.7",
```
Comment (nitsan) [inline package.json:3]: why you jump from 0.1.3 to 0.1.7?

File: package.json [hunk 2]
```diff
@@ -14,12 +14,13 @@
...
   },
-  "peerDependencies":{
+  "peerDependencies": {
     "mongoose": "*"
-},
+  },
   "dependencies": {
     "jasmine-node": "^1.14.5",
-    "lodash": "~3.10.0",
+    "lodash.clonedeep": "^4.5.0",
```
Comment (nitsan) [inline package.json:22]: there is only lodash.clonedeep and lodash.isequal in all this project?

**Review by nitsan — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (eran10) [reply package.json:22]: yes

File: package.json (hunk 1 above)
Comment (eran10) [reply package.json:3]: today the tag version is 0.1.6 and it always should be the same as latest, after i will release a new tag then it will be 0.1.7

**Review by nitsan — APPROVED** [review]

**Pull request merged**